### ISSUE TITLE
File refresh on close

### ISSUE THREAD

**Issue by afrogonabike** [notepad-plus-plus/notepad-plus-plus#1400]
This bug is minor but it annoys me on a daily basis.  I often have files open that are changed on disk by other applications.  I'm notified by Notepad++ when the app gets focus with a message window (and need to continue with this mode if it can be turned off, as it is useful for testing purposes).  The bug is that if I right click the the notepad++ icon on the taskbar (windows 10) and click close, if any file has changed the close is lost due to the message being shown.  You then have to click on the app, acknowledge the message and then close the app again.

Otherwise great app thanks!

**Comment by MetaChuh** [comment]
we are closing this issue due to multiple newer notepad++ versions since it was filed.

please update your notepad++ to the latest version [<kbd>Notepad++ 7.6.6</kbd>](https://notepad-plus-plus.org/download/v7.6.6.html) and migrate or reinstall your plugins.
if your issue still persists, and it is still of relevance to you, please feel free to file a new issue as detailed and reproducible as possible.

thank you for your help and best regards.

-----
```
general notes:

please close your issues as soon as you are satisfied, to help
keeping the notepad++ issue tracker clean.

when creating a new issue, please provide all information as shown at the issue template.
your notepad++ debug info is part of the minimum requirements.
(? > Debug Info... > Copy debug info into clipboard).

please also conduct a search at the issue tracker beforehand, to check if a similar issue
already exists.

for general questions, or if it is not sure, whether your issue is directly related to
the notepad++ source code, please visit us at the notepad++ community forum, and
search if related topics exists. you are welcome to post either at similar
topics, or to create a new topic at Help Wanted or General Discussion.
(no extra account is needed, just use your github account to sign in)
```
[<p align="center"><kbd> &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; Click here to visit the Notepad++ Community Forum &emsp; &emsp; &emsp; &emsp; &emsp; &emsp;&emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; </kbd></p>](https://notepad-plus-plus.org/community/)

**Comment by bartoli** [comment]
I see it fixed in 7.6.6 anyways ;-)
